cachi2/core/package_managers/rpm/main.py

```diff
@@ -320,7 +332,13 @@ def _download(lockfile: RedhatRpmsLock, output_dir: Path) -> dict[Path, Any]:
             }
             Path.mkdir(dest.parent, parents=True, exist_ok=True)
 
-        asyncio.run(async_download_files(files, get_config().concurrency_limit))
+        asyncio.run(
+            async_download_files(
+                files,
+                get_config().concurrency_limit,
+                ssl_context=_get_ssl_context(options=options.model_dump() if options else None),
```

Comment:
This should be `_get_ssl_context(options=options.model_dump()) if options else None` (mind the parenthesis after `dump`)